crates/aiken/src/cmd/tx/simulate.rs

```diff
@@ -85,6 +97,71 @@ pub fn exec(
 
     let tx = MultiEraTx::decode_for_era(Era::Conway, &tx_bytes).into_diagnostic()?;
 
+    let mut overrides: HashMap<ScriptHash, PlutusScript> = HashMap::new();
+
+    if !script_overrides.is_empty() {
+        with_project(None, false, true, false, |p| {
+            eprintln!(
+                "{} script overrides",
+                "      Computing"
+                    .if_supports_color(Stderr, |s| s.purple())
+                    .if_supports_color(Stderr, |s| s.bold()),
+            );
+            let hash_overrides: HashMap<ScriptHash, ScriptHash> = script_overrides
+                .iter()
+                .try_fold::<_, _, Result<_, aiken_project::error::Error>>(
+                    HashMap::new(),
+                    |mut acc, script_override| {
+                        let mut parts = script_override.split(":");
+
+                        let from = parts
+                            .next()
+                            .ok_or(Error::ScriptOverrideArgumentParseError {
+                                value: script_override.clone(),
+                            })?;
+
+                        let to = parts
+                            .next()
+                            .ok_or(Error::ScriptOverrideArgumentParseError {
+                                value: script_override.clone(),
+                            })?;
+
+                        let from_hash = ScriptHash::from(
+                            hex::decode(from)
+                                .map_err(|_| Error::ScriptOverrideArgumentParseError {
+                                    value: script_override.clone(),
+                                })?
+                                .as_slice(),
+                        );
```

Comment:
Repeating the `value` (provided by the user) in the error isn't going to be much useful to troubleshoot. Instead, it would be better to provide in the error:

1. The index (use: [`Ordinal.suffix`](https://docs.rs/ordinal/latest/ordinal/struct.Ordinal.html#method.suffix)) of the failing argument
2. A hand-written hint about the error (e.g. "couldn't find the left part of the mapping (from), did you provide colon-separated script hashes?", "couldn't parse the left part of the mapping: invalid hex encoding: {e}" ...)